pypeman/channels.py

```diff
@@ -395,6 +402,7 @@ async def handle(self, msg):
                         subchan_endnodes_fut.add_done_callback(self._reset_sub_chan_endnodes)
                         if self.wait_subchans:
                             await subchan_endnodes_fut
+                    self.logger.info("%s end handle %s", self, msg)
```

Comment:
`self` cannot always be pickled and not always be unpickled by a  socket loggers.

Migt be same issue for `msg`

Reply:
i replaced self (=chan) by str(self) and msg by str(msg)